src/main/java/com/aquaticinformatics/aquarius/sdk/samples/ServiceModel.java

```diff
@@ -1446,6 +1473,8 @@ public static class DeleteFieldVisits implements IReturnVoid
         public DeleteFieldVisits setSort(String value) { this.Sort = value; return this; }
         public Instant getStartStartTime() { return StartStartTime; }
         public DeleteFieldVisits setStartStartTime(Instant value) { this.StartStartTime = value; return this; }
+        public Instant getStartModificationTime() { return StartModificationTime; }
```

Comment:
as mentioned in https://github.com/AquaticInformatics/aquarius-sdk-java/pull/200 the service model update looks accurate to changes in Samples API over the last couple years.